app/controllers/users_controller.rb

```diff
@@ -28,54 +28,37 @@ def create
 
     respond_to do |format|
       if @user.save
-        format.html { redirect_to @user, notice: 'User was successfully created.' }
-        format.json { render :show, status: :created, location: @user }
+        redirect_to @user, notice: 'User was successfully created.'
       else
-        format.html { render :new }
-        format.json { render json: @user.errors, status: :unprocessable_entity }
+        render :new
       end
     end
   end
 
-  # PATCH/PUT /users/1
-  # PATCH/PUT /users/1.json
   def update
     respond_to do |format|
       if @user.update(user_params)
-        format.html { redirect_to @user, notice: 'User was successfully updated.' }
-        format.json { render :show, status: :ok, location: @user }
+        redirect_to @user, notice: 'User was successfully updated.'
       else
-        format.html { render :edit }
-        format.json { render json: @user.errors, status: :unprocessable_entity }
+        render :edit
       end
     end
   end
 
-  # DELETE /users/1
-  # DELETE /users/1.json
   def destroy
     @user.destroy
-    respond_to do |format|
-      format.html { redirect_to users_url, notice: 'User was successfully destroyed.' }
-      format.json { head :no_content }
-    end
+    redirect_to users_url, notice: 'User was successfully destroyed.'
   end
 
   private
     # Use callbacks to share common setup or constraints between actions.
-    def set_user
-      @user = User.find(params[:id])
-    end
+  def set_user
+    @user = User.find(params[:id])
+  end
 
     # Never trust parameters from the scary internet, only allow the white list through.
-    def user_params
-      params.require(:user).permit(:name, :email, :phone, :spire, :active, :admin)
-    end
-  def require_admin_in_roster_or_self
-    return if @current_user == @user || @current_user.admin_in?(@roster)
-    # ... and return is correct here
-    # rubocop:disable Style/AndOr
-    head :unauthorized and return
-    # rubocop:enable Style/AndOr
+  def user_params
+    params.require(:user).permit(:name, :email, :phone, :spire, :active, :admin)
```

Comment:
If I'm not an admin, can I change whether or not I'm an admin by injecting `admin: true` while changing my email or something?

If that's out of scope now, that's fine.